rb_ws/src/buggy/scripts/auton/autonsystem.py

```diff
@@ -82,9 +84,6 @@ def __init__(self,
         self.steer_publisher = rospy.Publisher(
             self_name + "/buggy/input/steering", Float64, queue_size=1
         )
-        self.brake_publisher = rospy.Publisher(
-            self_name + "/buggy/input/brake", Float64, queue_size=1
-        )
```

Comment:
Why was brake publisher removed, are we not using this anymore?

Reply:
we have never really used this, commands sent to firmware gets ignored and gets overidden. IMO we should leave it in though because it'll get logged still.